api/users/urls.py

```diff
@@ -13,4 +13,5 @@
     url(r'^(?P<user_id>\w+)/institutions/$', views.UserInstitutions.as_view(), name=views.UserInstitutions.view_name),
     url(r'^(?P<user_id>\w+)/registrations/$', views.UserRegistrations.as_view(), name=views.UserRegistrations.view_name),
     url(r'^(?P<user_id>\w+)/relationships/institutions/$', views.UserInstitutionsRelationship.as_view(), name=views.UserInstitutionsRelationship.view_name),
+    url(r'^(?P<user_id>\w+)/public_files/$', views.UserPublicFiles.as_view(), name=views.UserPublicFiles.view_name),
```

Comment:
I'm not in love with this route, but that part is up to @brianjgeiger 

I'd suggest, as an alternative, something like `/files/public/<user_id>/` to return a list of serialized public files for that user, rather than returning a nonstandard serialization of a node.